deploy/kubeturbo-operator/deploy/crds/charts_v1alpha1_kubeturbo_crd.yaml

```diff
@@ -115,12 +122,18 @@ spec:
                 kubeletport:
                   description: Identify kubelet port
                   type: integer
-                stitchuuid:
-                  description: Identify if using uuid or ip for stitching
-                  type: boolean
                 sccsupport:
                   description: Allow kubeturbo to execute actions in OCP
                   type: string
+                failVolumePodMoves:
+                  description: Allow kubeturbo to reschedule pods with volumes attached
+                  type: string
+                busyboxExcludeNodeLabels:
+                  description: Do not run busybox on these nodes to discover the cpu frequency with k8s 1.18 and later, default is kubernetes.io/os=windows,beta.kubernetes.io/os=windows
```

Comment:
s/`default is kubernetes.io/os=windows,beta.kubernetes.io/os=windows`/`default is either of kubernetes.io/os=windows or beta.kubernetes.io/os=windows present as node label`

Reply:
done, thanks